frontend/__tests__/services/wsaddress-service.server.test.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint @typescript-eslint/no-unused-vars: ["error", { "varsIgnorePattern": "^_" }] */
+import { describe, expect, it } from 'vitest';
+import { ParseWSAddressResponseDTO } from '~/dtos/parse-address-response-dto.server';
+import { ValidateWSAddressResponseDTO } from '~/dtos/validate-address-response-dto.server';
+import { server } from '~/mocks/node';
```

Comment:
Unit test mocks should be isolated and laser-focused on the unit being tested.

The mocks that exist in `~/mocks/node` are intended to be used for mocking during development (at runtime). For a unit test, you should define your own set of mocks specifically for your tests.

Reply:
Created separate wsaddress MSW

Reply:
Sorry, I don't think I was clear in my previous comment. My suggestion was to move the mocks inside of `__tests__/services/wsaddress-service.server.test.ts`, where they can be used to test the specific scenarios.

This way, you can ensure that your tests are based on well-defined and unambiguous behavior (e.g., "*when WSAddress returns X, we should do Y*"). This also helps to avoid breaking the tests by changing the application-scoped mocks in `app/mocks/`.

In my opinion, tests should have clear and explicit setup code that shows exactly what situation is being tested, and this is not possible if the mocks are outside of the test.

Reply:
Moved wsaddress msw into test file